src/test/java/com/kodilla/ecommercee/domain/GroupTest.java

```diff
@@ -0,0 +1,72 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GroupTest {
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @After
+    public void clearRepository(){
+        groupRepository.findAll().clear();
+    }
+
+    @Test
+    public void testSave() {
+        //Given
+        Group group = new Group(1L, "Chemia");
```

Comment:
Użyłbym raczej:
```suggestion
        Group group = new Group(null, "Chemia");
```
Choć efekt będzie dokładnie taki sam.